spotlight-client/src/charts/WindowedTimeSeries.tsx

```diff
@@ -103,7 +104,14 @@ const WindowedTimeSeries: React.FC<{
   defaultRangeEnd: Date;
   defaultRangeStart?: Date;
   setTimeRangeId: (id: WindowSizeId) => void;
-}> = ({ data, defaultRangeEnd, defaultRangeStart, setTimeRangeId }) => {
+  preview?: boolean;
```

Comment:
as I have commented elsewhere, I'm not really a fan of "preview" in this interface, which is meant to be more generic. I think if we instead had props for like, `showMinimap` and `showLegend`, then the parent could pass those as needed to create the desired "preview" effect.